packages/react-components/src/components/PromoBanner.spec.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import { render, fireEvent } from '../test-utils';
+import { PromoBanner, PromoBannerSize } from './PromoBanner';
+
+const customClass = 'my-css-class';
+const promoHeader = 'This example headline has 40 characters';
+const promoContent = 'This example content';
+const text = 'Example text';
+const img = 'https://via.placeholder.com/100';
+const mockedFunction = jest.fn();
+
+describe('<PromoBanner> component', () => {
+  it('should allow for custom class', () => {
+    const { container } = render(
+      <PromoBanner header={promoHeader} className={customClass}>
+        {promoContent}
+      </PromoBanner>
+    );
+
+    expect(container.firstChild).toHaveClass(customClass);
+  });
+
+  it('should display button', () => {
+    const { getByTestId } = render(
+      <PromoBanner header={promoHeader} buttonText={text}>
+        {promoContent}
+      </PromoBanner>
+    );
+
+    expect(getByTestId('button')).toBeTruthy();
+  });
+
+  it('should display link', () => {
+    const { getByTestId } = render(
+      <PromoBanner header={promoHeader} linkText={text}>
+        {promoContent}
+      </PromoBanner>
+    );
+
+    expect(getByTestId('link')).toBeTruthy();
```

Comment:
The same with link. I would check if `linkText` is rendered under button (which also makes me confused - why is it called a `linkText`?